### PR TITLE
[FEAT] sync typescript versions

### PR DESCRIPTION
**Pull request by kikoruiz**
Sync all the TypeScript versions along our SUI tools.

### CODE REVIEW

File: packages/sui-mock/package.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
   "name": "@s-ui/mock",
-  "version": "1.2.0",
+  "version": "1.3.0-beta.3",
```
Comment (kikoruiz) [inline packages/sui-mock/package.json:3]: Will be removed before merging.

File: packages/sui-test-contract/src/setup/utils.js [hunk 2]
```diff
@@ -61,7 +61,7 @@ export const writerFactory = providers => (path, data) => {
   })
 
   console.log(`Writing the Pact file "${path}"`) // eslint-disable-line
-  writeData2File(path, data)
+  createWriter()(path, data)
```
Comment (kikoruiz) [inline packages/sui-test-contract/src/setup/utils.js:64]: Fix for a breaking change introduced in [v1.4.0](https://github.com/pactflow/pact-msw-adapter/releases/tag/v1.4.0): https://github.com/pactflow/pact-msw-adapter/commit/6d4554158d4ef2671e88e556604319379659b5a2

**Review by carlosvillu — APPROVED** [review]

File: packages/sui-test-contract/package.json [hunk 3]
```diff
@@ -17,8 +17,8 @@
   },
   "dependencies": {
     "@pact-foundation/pact": "9.18.1",
-    "@pactflow/pact-msw-adapter": "1.2.1",
-    "@s-ui/mock": "1",
+    "@pactflow/pact-msw-adapter": "1.5.0",
+    "@s-ui/mock": "beta",
```
Comment (kikoruiz) [inline packages/sui-test-contract/package.json:21]: We'll change it when the new `sui-mock` version is released in the next master workflow execution.

**Review by andresz1 — APPROVED** [review]

**Review by nucliweb — APPROVED** [review]

**Review by carlosvillu — APPROVED** [review]

**Comment by kikoruiz** [comment]
It won't be fixed here, it will be done in: https://github.com/SUI-Components/sui/pull/1614

**Pull request closed without merging**